### ISSUE TITLE
Block everything but sweden, but still put in finnish servers?

### ISSUE THREAD

**Issue by wazerstar** [FN-FAL113/cs2-server-picker#39]
I could tell from the ping and the status print in console i was routed to helsinki, what can I do more?

de_vertigo-Valve_Counter-Strike_2_helsinki_Server__srcds1033-helxx_.dem

**Comment by FN-FAL113** [comment]
did u check steam overlay by doing a shift tab, before u accept any match you can check through steam overlay what server you got routee to

**Comment by wazerstar** [comment]
> do you also have any third party antivirus, blocking every server and leaving out one is not recommended as stated in the readme FAQ.

I checked and it says helsinski and also in console. 

I have been trying to block these so far.
```


162.254.198.101
162.254.198.102

155.133.252.37
155.133.252.38
155.133.252.53
155.133.252.84
155.133.252.85
155.133.252.86
155.133.252.87
155.133.252.88
155.133.252.89
```

Also

The way I can see if its from helsinski is also by the ping, I have normal ping 5-13 to sweden.

To finland its always 20 or more and the console outputs 13+7 = 20 ping.

```
[SteamNetSockets] [#2812326370 SDR server steamid:90181615416427537 vport 0] Requesting session from sto#165 (162.254.198.101:27041).  Ping = 20 = 13+7 (front+back).  Rank=1 Score=33
[SteamNetSockets] [#2812326370 SDR server steamid:90181615416427537 vport 0] Selecting sto#165 (162.254.198.101:27041) as primary.  (Ping = 20 = 13+7+0 (front+interior+remote).  Rank=1 Score=21)
[SteamNetSockets] [#2812326370 SDR server steamid:90181615416427537 vport 0] Requesting session from sto2#204 (155.133.252.84:27034).  Ping = 20 = 13+7 (front+back).  Rank=2 Score=33
[SteamNetSockets] [#2812326370 SDR server steamid:90181615416427537 vport 0] Selecting sto2#204 (155.133.252.84:27034) as backup #1 (Ping = 20 = 13+7+0 (front+interior+remote).  Rank=1 Score=21)
```

I tried to block all these so far.

I dont have any antivirus installed, my windows OS is slimmed down to minimum resources.

**Comment by FN-FAL113** [comment]
> > do you also have any third party antivirus, blocking every server and leaving out one is not recommended as stated in the readme FAQ.
> 
> I checked and it says [helsinski](https://i.imgur.com/ybrUDra.png) and also in console, and you cannot see the ip in the overlay
> 
> I have been trying to block these so far, those are the ones that have been posted in console.
> 
> ```
> 
> 
> 162.254.198.101
> 162.254.198.102
> 
> 155.133.252.37
> 155.133.252.38
> 155.133.252.53
> 155.133.252.84
> 155.133.252.85
> 155.133.252.86
> 155.133.252.87
> 155.133.252.88
> 155.133.252.89
> ```
> 
> Also
> 
> The way I can see if its from helsinski is also by the ping, I have normal ping 5-13 to Sweden.
> 
> To Finland its always 20 or more and the console outputs 13+7 = 20 ping.
> 
> ```
> [SteamNetSockets] [#2812326370 SDR server steamid:90181615416427537 vport 0] Requesting session from sto#165 (162.254.198.101:27041).  Ping = 20 = 13+7 (front+back).  Rank=1 Score=33
> [SteamNetSockets] [#2812326370 SDR server steamid:90181615416427537 vport 0] Selecting sto#165 (162.254.198.101:27041) as primary.  (Ping = 20 = 13+7+0 (front+interior+remote).  Rank=1 Score=21)
> [SteamNetSockets] [#2812326370 SDR server steamid:90181615416427537 vport 0] Requesting session from sto2#204 (155.133.252.84:27034).  Ping = 20 = 13+7 (front+back).  Rank=2 Score=33
> [SteamNetSockets] [#2812326370 SDR server steamid:90181615416427537 vport 0] Selecting sto2#204 (155.133.252.84:27034) as backup #1 (Ping = 20 = 13+7+0 (front+interior+remote).  Rank=1 Score=21)
> ```
> 
> I tried to block all these so far.
> 
> I dont have any antivirus installed, my windows OS is slimmed down to minimum resources.
> 
> I'm wondering if I should just do a 65.109.0.0/24 and be over with it..

162.254.198.* are Stockholm - Kista (Sweden) while
155.133.252.* are Stockhold - Bromma (Sweden) servers

**Comment by FN-FAL113** [comment]
Pretty much the console IP's are from sweden, you're just being connected to a relay which does not expose its real ip address.

**Comment by FN-FAL113** [comment]
> > do you also have any third party antivirus, blocking every server and leaving out one is not recommended as stated in the readme FAQ.
> 
> I checked and it says [helsinski](https://i.imgur.com/ybrUDra.png) and also in console, and you cannot see the ip in the overlay
> 
> I have been trying to block these so far, those are the ones that have been posted in console.
> 
> ```
> 
> 
> 162.254.198.101
> 162.254.198.102
> 
> 155.133.252.37
> 155.133.252.38
> 155.133.252.53
> 155.133.252.84
> 155.133.252.85
> 155.133.252.86
> 155.133.252.87
> 155.133.252.88
> 155.133.252.89
> ```
> 
> Also
> 
> The way I can see if its from helsinski is also by the ping, I have normal ping 5-13 to Sweden.
> 
> To Finland its always 20 or more and the console outputs 13+7 = 20 ping.
> 
> ```
> [SteamNetSockets] [#2812326370 SDR server steamid:90181615416427537 vport 0] Requesting session from sto#165 (162.254.198.101:27041).  Ping = 20 = 13+7 (front+back).  Rank=1 Score=33
> [SteamNetSockets] [#2812326370 SDR server steamid:90181615416427537 vport 0] Selecting sto#165 (162.254.198.101:27041) as primary.  (Ping = 20 = 13+7+0 (front+interior+remote).  Rank=1 Score=21)
> [SteamNetSockets] [#2812326370 SDR server steamid:90181615416427537 vport 0] Requesting session from sto2#204 (155.133.252.84:27034).  Ping = 20 = 13+7 (front+back).  Rank=2 Score=33
> [SteamNetSockets] [#2812326370 SDR server steamid:90181615416427537 vport 0] Selecting sto2#204 (155.133.252.84:27034) as backup #1 (Ping = 20 = 13+7+0 (front+interior+remote).  Rank=1 Score=21)
> ```
> 
> I tried to block all these so far.
> 
> I dont have any antivirus installed, my windows OS is slimmed down to minimum resources.
> 
> I'm wondering if I should just do a 65.109.0.0/24 and be over with it..

Also this app is tested with microsoft defender that manages the windows firewall.

**Comment by wazerstar** [comment]
> > > do you also have any third party antivirus, blocking every server and leaving out one is not recommended as stated in the readme FAQ.
> > 
> > 
> > I checked and it says [helsinski](https://i.imgur.com/ybrUDra.png) and also in console, and you cannot see the ip in the overlay
> > I have been trying to block these so far, those are the ones that have been posted in console.
> > ```
> > 
> > 
> > 162.254.198.101
> > 162.254.198.102
> > 
> > 155.133.252.37
> > 155.133.252.38
> > 155.133.252.53
> > 155.133.252.84
> > 155.133.252.85
> > 155.133.252.86
> > 155.133.252.87
> > 155.133.252.88
> > 155.133.252.89
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Also
> > The way I can see if its from helsinski is also by the ping, I have normal ping 5-13 to Sweden.
> > To Finland its always 20 or more and the console outputs 13+7 = 20 ping.
> > ```
> > [SteamNetSockets] [#2812326370 SDR server steamid:90181615416427537 vport 0] Requesting session from sto#165 (162.254.198.101:27041).  Ping = 20 = 13+7 (front+back).  Rank=1 Score=33
> > [SteamNetSockets] [#2812326370 SDR server steamid:90181615416427537 vport 0] Selecting sto#165 (162.254.198.101:27041) as primary.  (Ping = 20 = 13+7+0 (front+interior+remote).  Rank=1 Score=21)
> > [SteamNetSockets] [#2812326370 SDR server steamid:90181615416427537 vport 0] Requesting session from sto2#204 (155.133.252.84:27034).  Ping = 20 = 13+7 (front+back).  Rank=2 Score=33
> > [SteamNetSockets] [#2812326370 SDR server steamid:90181615416427537 vport 0] Selecting sto2#204 (155.133.252.84:27034) as backup #1 (Ping = 20 = 13+7+0 (front+interior+remote).  Rank=1 Score=21)
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > I tried to block all these so far.
> > I dont have any antivirus installed, my windows OS is slimmed down to minimum resources.
> > I'm wondering if I should just do a 65.109.0.0/24 and be over with it..
> 
> Also this app is tested with microsoft defender that manages the windows firewall. I'm not quite sure about different flavors of windows that are trimmed down.

Im using my firewall for other things its working fine.

I tested several apps and website, its working as intended, some ips to finland is missing in the app, what external apps can  test to see which ip I get ?

**Comment by FN-FAL113** [comment]
please refer to FAQ number 1. We have no access to the actual ip addresses of the game servers but only to relays so you can block helsinki relays from the app.

**Comment by FN-FAL113** [comment]
> > please refer to FAQ number 1. We have no access to the actual ip addresses of the game servers but only to relays so you can block helsinki relays from the app.
> > ![image](https://private-user-images.githubusercontent.com/88238718/305957870-ca3408f3-a48f-4684-8f0e-19db27e5bc28.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgzNDgxNDQsIm5iZiI6MTcwODM0Nzg0NCwicGF0aCI6Ii84ODIzODcxOC8zMDU5NTc4NzAtY2EzNDA4ZjMtYTQ4Zi00Njg0LThmMGUtMTlkYjI3ZTViYzI4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE5VDEzMDQwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYxZTdmZDdjNmViMjhlZjZhYmE5ZTIwYWYxNDhlNmI4NTgyZjgyYWM4ZTkwODZjOWY3MDE1M2QxMTI0ODYwY2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MikJuXGWqlYe7_0zJyMazyoyHzHJ6SK4umghJb75hoE)
> 
> It's already blocked, but ip needs an update, you cannot see the ip via overlay in-game..
> 
> https://i.imgur.com/TJX3Nu5.png

you can check the server location via overlay in-game, please leave out some servers unblocked for optimize relaying of connections. The ip addresses are synchronized dynamically when there are updates. Check out this video by someone who used the app, they opened the overlay to check what server they got connected to, the ip address is not public there but its a great way to test:
https://www.youtube.com/watch?v=A2BPAp4QrUA
https://www.youtube.com/watch?v=54aqpRMcyPo

**Comment by wazerstar** [comment]
> > > please refer to FAQ number 1. We have no access to the actual ip addresses of the game servers but only to relays so you can block helsinki relays from the app.
> > > ![image](https://private-user-images.githubusercontent.com/88238718/305957870-ca3408f3-a48f-4684-8f0e-19db27e5bc28.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgzNDgxNDQsIm5iZiI6MTcwODM0Nzg0NCwicGF0aCI6Ii84ODIzODcxOC8zMDU5NTc4NzAtY2EzNDA4ZjMtYTQ4Zi00Njg0LThmMGUtMTlkYjI3ZTViYzI4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE5VDEzMDQwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYxZTdmZDdjNmViMjhlZjZhYmE5ZTIwYWYxNDhlNmI4NTgyZjgyYWM4ZTkwODZjOWY3MDE1M2QxMTI0ODYwY2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MikJuXGWqlYe7_0zJyMazyoyHzHJ6SK4umghJb75hoE)
> > 
> > 
> > It's already blocked, but ip needs an update, you cannot see the ip via overlay in-game..
> > https://i.imgur.com/TJX3Nu5.png
> 
> you can check the server location via overlay in-game, please leave out some servers unblocked for optimize relaying of connections. The ip addresses are synchronized dynamically when there are updates. Check out this video by someone who used the app, they opened the overlay to check what server they got connected to, the ip address is not public there but its a great way to test: https://www.youtube.com/watch?v=A2BPAp4QrUA https://www.youtube.com/watch?v=54aqpRMcyPo

I'm not sure if you are understanding me?

I already posted the image to you to show you the overlay only shows that country/city you are routed to and NOT the ip, I'm not sure why you are posting guides to the tool I have already done and is working.

Why would I unblock anything but Sweden when its Swedish servers I want to play on and nothing else :)

Its working fine besides I sometimes get routed to Helsinski(Finland)




I'm more interested into how to find those ip like the tool has for Finland?

![4XcLPw6](https://github.com/FN-FAL113/cs2-server-picker/assets/4474519/990c7968-98a2-4079-bd72-1ee58923eba1)

**Comment by FN-FAL113** [comment]
The ip you see from the firewall is what it is, I've also mentioned many times to read the FAQ from this project readme. I cannot help any further in regards to finding ip that the game does not expose.

**Comment by FN-FAL113** [comment]
I'll be closing this for now, you may keep commenting here if you want.

**Comment by FN-FAL113** [comment]
The ip addresess the app blocks is what valve is able to expose publicly which are relays, they are synced through a web resource api. T he app will show a message if there are any updates to the server ip addresses, the real ip addresses to the game servers are not exposed to prevent ddosing. If you think this app is missing any ip addresses that you see ingame please drop it here, I don't have the game anymore installed currently my laptop cannot handle it due to lacking of a dedicated gpu.
